writing/kill-the-dog.md

```diff
@@ -0,0 +1,25 @@
+# Killing the Dog - My new reason for practicing mindfulness
+
+Anyone else seriously wanted to kill their dog ☠️🐶? And gone through a bunch of plans to make it seem like an accident?
+
+- "Accidentally" leaving the front door open so she runs into the road
+- Host a house party 🥳, put on "Who let the dogs out", turn it up and keep putting the puppy in the middle of the dance floor until someone crushes her to death with their funky moves
+- Travel back to the Roman times, tell everyone the dog is the Messiah and wait until the inevitable crucifixion ✟
+
+Yeah, I thought about this. In the first few weeks of getting the most adorable little puppy I found myself in a permanent state where I couldn't think clearly for the first time in my life. Possibly due to a mixture of sleep deprivation, constant distraction, worrying and having massively underestimated the commitment.
+
+Either way it scared the shit 💩 out of me, and I became super unhappy. To a point where I was seriously considering how to kill the dog and make it look like an accident.
+
+It made me sympathise more with people who are aware that they are losing their minds. And having lost my Grandma to Alzheimers a year or so before, the scary reality of this problem was very prominent in my mind.
+
+So I've been wondering about how you could prepare yourself to better deal with future situations like this. And have a theory for ya'll:
+
```

Comment:
I want to suggest you add a couple of words after "future situations" instead of "like this."  On first reading, I couldn't figure out what situation you meant, and actually at first thought you meant getting a dog.